src/utils/fmt.zig

```diff
@@ -154,19 +154,19 @@ inline fn boundedStringMaxArg(comptime T: type) ?T {
 /// Tries to format the real path resolved from `dir` and `pathname`.
 /// Should it encounter an error when doing so, `"(error.Name)/pathname"`
 /// is printed instead.
-pub inline fn tryRealPath(dir: std.fs.Dir, pathname: []const u8) TryRealPathFmt {
-    return .{
+pub inline fn tryRealPath(dir: std.fs.Dir, pathname: []const u8) std.fmt.Formatter(TryRealPathCtx.realPathFormat) {
```

Comment:
I guess there's no difference, I'm just more familiar with using `std.fmt.Formatter` since it's more familiar. Don't really have an opinion if this gets in or not.

Reply:
It seems that `std.fmt.Formatter` exists solely to reduce boilerplate, but in this case we're implementing the struct either way, which defeats the purpose.

I can see that it helps communicate intent to those familiar with it, but it also adds complexity for others who will need to navigate extra code. I lean towards the simpler approach, but I'm fine either way.